microprofile/WFLY-12737_EAP7-1368_quickstart-for-mp-health-2.1.adoc

```diff
@@ -0,0 +1,62 @@
+= WFLY-12737 / EAP7-1368 Quickstart for MP Health 2.1
```

Comment:
FYI we don't need to include the EAP7 issue in the file name or title, it can be listed in the related issues section instead.

As this is a community project those monitoring the project may not have access to the EAP7 issues as they are internal planning issues.

Reply:
I agree - maybe in the issue section the community one should be listed first and maybe add a suffix to the EAP7 Jira something like `(internal)` so people aren't confused why they cannot access things in an open procesS?